.env-wb-garbage-collector

```diff
@@ -20,7 +20,7 @@ WEBSERVER_LOGIN=null
 #WEBSERVER_RESOURCE_MANAGER from .env
 WEBSERVER_SCICRUNCH=null
 WEBSERVER_STATICWEB=null
-WEBSERVER_STORAGE=null
+# WEBSERVER_STORAGE=null

```

Comment:
This change *enables* the storage plugin, correct? If true, I find this highly non-obvious...

Reply:
yes